src/html/shared/header.html

```diff
@@ -1,23 +1,21 @@
-<header class="c-header">
+<nav class="c-header  is-scroll" style="background-color:white;">
```

Comment:
Apply `background-color: white` to your css instead of inline styling it.

Also, `is-scroll` doesn't really explain the state of your nav header.

Looking at the behaviour cases of your header component. You have

1. Navigation that is sticky and on top of a hero (transparent)
2. Navigation that is sticky and **not** on top of a hero (not transparent)

So your navigation needs to cater for 

1. Background transparent when in the hero region
2. Background white when you start scrolling (maybe after 200px - javascript implementation)

So the variable that changes is the transparency of your navigation, which in this case would be better to go with `is-transparent`. Explains alot more!